conceptnet5/api.py

```diff
@@ -182,6 +182,9 @@ def query_similarity(node1, node2):
     Query for the similarity between node1 and node2. Return the cosine
     similarity between the vectors of these two terms.
     """
+    if node1 is None or node2 is None:
+        return error({}, '400', 'Arguments should be called node1 and node2.')
```

Comment:
It seems wrong for it to be the string `'400'` in one case and the integer 400 in the other.